utils/file_operations.py

```
@@ -0,0 +1,285 @@
+                continue
+
+            file_path = save_path / name
+            if file_path.exists():
+                file_paths.append(file_path)
+            else:
+                logging.debug(f"File does not exist (may have been moved): {file_path}")
+
+        return file_paths
+
+    except Exception as e:
+        logging.exception(f"Error getting file paths for torrent {torrent_hash}: {e}")
```

Ruff (0.14.6)

Redundant exception object included in `logging.exception` call

(TRY401)